communitiesuk/funding-service-design-fund-application-builder
# FS-4766 - Update feedback survey config

### Ticket
[Update Feedback survey config options  to Checkbox from Json field](https://mhclgdigital.atlassian.net/browse/FS-4766)

### Change description

- Displaying error message if fund not specified when creating round (bug not specific to ticket)
- Including research survey fields in feedback survey config default and test data
- Breaking out feedback survey config into separate radio fields on the frontend (thanks @NarenderRajuB for the suggestion)

### How to test
Go to `/round` and fill out form to create a new round. Note new radio fields including:

- Has feedback survey
- Has section feedback
- Has research survey
- Is feedback survey optional
- Is section feedback optional
- Is research survey optional

Select randomly between "Yes" and "No" for each.

Submit the form. Form should be submitted successfully.

View the round and check values for radio fields match input (not sure how to do this @Adam-W1 ?)

### Screenshots of UI changes (if applicable)
![image](https://github.com/user-attachments/assets/968f832d-77b5-493a-98dc-4543bd1168b1)
